modules/ivc_champva/app/models/ivc_champva/vha_10_10d.rb

```diff
@@ -48,8 +48,14 @@ def add_applicant_properties
       applicants = @data['applicants']
```

Comment:
Please have the models inherit from a common base to reduce duplication (DRY).

Reply:
@dellerbie Please create a ticket. Conducting extensive code reviews to fix multiple files each time can be cumbersome and difficult to maintain. I attached a document to help DRY up those models.
[CHAMPVA_DRY_Refactoring_Complete (1).md](https://github.com/user-attachments/files/23612692/CHAMPVA_DRY_Refactoring_Complete.1.md)